scripts/jobs/housing/housing_apply_gx_dq_tests.py

```diff
@@ -0,0 +1,127 @@
+import awswrangler as wr
+from datetime import datetime
+import json
+import logging
+import sys
+
+from awsglue.utils import getResolvedOptions
+import great_expectations as gx
+import pandas as pd
+from pyathena import connect
+from scripts.helpers.housing_gx_dq_inputs import gx_dq_housing_config, table_list
+
+
+args = getResolvedOptions(sys.argv,
+                          ['region_name',
+                           's3_endpoint',
+                           's3_target_location',
+                           's3_staging_location',
+                           'target_database',
+                           'tables_list',
+                           'gx_docs_bucket',
+                           'gx_docs_prefix'])
+
+region_name = args['region_name']
+s3_endpoint = args['s3_endpoint']
+s3_target_location = args['s3_target_location']
+gx_docs_bucket = args['gx_docs_bucket']
+gx_docs_prefix = args['gx_docs_prefix']
+s3_staging_location = args['s3_staging_location']
+target_database = args['target_database']
+target_table = args['target_table']
```

Comment:
Using the locals().update() method, which directly assigns the args dictionary keys as local variables, below is the same as your code.
```
arg_keys = [
    'region_name', 's3_endpoint', 's3_target_location', 's3_staging_location',
    'target_database', 'target_table', 'gx_docs_bucket', 'gx_docs_prefix'
]
args = getResolvedOptions(sys.argv, arg_keys)
# Update local variables dynamically using dictionary unpacking
locals().update(args)
```

This [URL ](https://www.quora.com/Given-a-dictionary-can-one-automatically-make-variables-out-of-their-key-value-pairs-in-Python) explains how it works (see the most-voted answer).